tests/replication.rs

```diff
@@ -435,6 +436,75 @@ fn update_replication() {
     assert!(component.0);
 }
 
+#[test]
+fn update_and_removal_with_stale_ack_replication() {
+    let mut server_app = App::new();
+    let mut client_app = App::new();
+    for app in [&mut server_app, &mut client_app] {
+        app.add_plugins((
+            MinimalPlugins,
+            ReplicationPlugins.set(ServerPlugin {
+                tick_policy: TickPolicy::EveryFrame,
+                ..Default::default()
+            }),
+        ))
+        .replicate::<BoolComponent>();
+    }
+
+    // 1. Connect a client.
```

Comment:
Could you please update comments to be more consistent with the rest of the codebase?
I.e. remove numbers, `-` and do not comment trivial stuff, like `despawn`, only when an explanation comes after. Just a personal preference.

Reply:
I'm not suggesting to remove the comments. Just adjust their styling. Numbers and `-` is pure style. We either re-do it everywhere like this, or stick to the current style.
And trivial comments, like "Update the client" doesn't add any new information to the code, I prefer to write comments for explanations or non-trivial things.

Reply:
Thanks! It's a matter of preference.
For example, writing the following in each test would be a huge duplication:
```rust
/// Connect client.
connect::single_client(&mut server_app, &mut client_app);
```
The code is explicit about what it does. And it's validated by compiler, while comments could be outdated.
But sometimes comments are useful.